### ISSUE TITLE
How to include a newline on the embed description?

### ISSUE THREAD

**Issue by faloi** [tsickert/discord-webhook#140]
Hi there, thanks for this handy action.

I've been unsuccessfully trying to add a new line, this is what I tried:

```yaml
embed-description: 'Ultimos cambios:\n_${{ github.event.head_commit.message }}_'
```

But sadly, the `\n` appears on the message: 

![image](https://user-images.githubusercontent.com/1585835/218850255-adae5d8e-f33e-4da7-bb47-3ec38b135847.png)

Any idea of how to achieve this?

**Comment by tsickert** [comment]
Glad it's helpful! 

I haven't confirmed this works, but you could this multiline yaml format with the `|`?

```yaml
  embed-description: |
    Ultimos cambios:
    ${{ github.event.head_commit.message }}
```

Hopefully the commit message will still interpolate with that format.

**Comment by faloi** [comment]
@tsickert worked like a charm! If you want to, I can send a pull request adding an example on the README.

BTW, any plans for supporting embed fields? It would be really nice to be able to include a list of fields.

**Comment by tsickert** [comment]
Glad it worked! A PR would be great if you'd like to open one. 

Embed fields are certainly doable. I looked at adding embed fields previously and I ended up not doing it because a message being able to have 10 embeds and then 20 fields per embed seemed like a scale issue. I think 9/10 use cases only have one embed though, so I can go back and add them since I know there's a need now. In theory, I can make fields supported for all embeds in a message, but I need to see if there's a limit of inputs per action--otherwise it's really more of a documentation issue than anything.
